### DISCUSSION TITLE
ESP32 Enable / Disable Bluetooth

### DISCUSSION THREAD

**Discussion by jhsa** [tttapa/Control-Surface#701] [General]
Is there some kind of command to enable or disable Bluetooth on ESP32 by pressing a button for example?

Thanks

João

**Comment by tttapa** [comment]
There's no Control Surface-specific command, but I'm sure it can be done using the ESP32 BLE API, I included some references to the docs and examples here: https://github.com/tttapa/Control-Surface/tree/master/src/MIDI_Interfaces/BLEMIDI/ESP32

**Comment by jhsa** [comment]
Checking, thanks.
